config/default/package.json

```diff
@@ -12,7 +12,7 @@
     "test-tasks": "npm run eslint && mocha --reporter progress test/tasks/*.spec.js --timeout 10000",
     "test-targets": "npm run eslint && mocha --reporter progress test/targets/*.spec.js --timeout 10000",
     "test-contact-summary": "npm run eslint && mocha --reporter progress test/contact-summary/*.spec.js --timeout 10000",
-    "test-unit": "mocha --recursive --reporter spec test --timeout 20000",
+    "test-unit": "TZ=Africa/Nairobi mocha --recursive --reporter spec test --timeout 20000",
```

Comment:
Tests that would fail on master passed on this branch only thanks to this addition and it wasn't obvious why at first.
So far, your solution is [what they recommend](https://github.com/sinonjs/fake-timers/issues/342#issuecomment-729495283) for our use case. We would need to add the environment variable to every other `test-*` script though.  
Since there seems to be no way to specify the timezone when mocking time in Sinon, could we make it clearer why we're using this environment variable?

Reply:
i've added the TZ into package.json but not sure where to specify why it is required... ideally we fix https://github.com/medic/cht-core/issues/6504

Reply:
I'm not sure what would be the best place either but I completely agree with you, the ideal solution is to fix the underlying bug :+1: